src/Frames/Joins.hs

```diff
@@ -0,0 +1,191 @@
+{-# LANGUAGE ConstraintKinds, DataKinds, FlexibleContexts, FlexibleInstances,
```

Comment:
We need a module-level comment. It'd be great to cater to those familiar with SQL with a brief SQL translation, and to general Haskell programmers by, ideally, putting an example into English words.

Reply:
Definitely. Will add.